project.clj

```diff
@@ -100,6 +100,7 @@
 
   :profiles {:dev {:source-paths  ["dev"]
                    :dependencies  [[org.clojure/tools.namespace "0.2.5"]
+                                   [org.clojure/java.jmx "0.2.0"]
```

Comment:
I tried rebuilding the tk-jetty9 testutils jar with this added to the tk-jetty9 `project.clj`:

~~~~
:testutils {:dependencies [[org.clojure/java.jmx "0.2.0"]]
               ...}
~~~~

The testjar built and installed to my local .m2 repository okay.  When I bumped my Puppet Server project.clj to use that latest snapshot install, though, I got the same "Could not locate clojure/java/jmx" error as before.

Is what I did what you were expecting to be able to do?